src/Classes/Particle/Connecter.ts

```diff
@@ -0,0 +1,98 @@
+import {Particle} from "../Particle";
+import {Container} from "../Container";
+import {IRgb} from "../../Interfaces/IRgb";
+import {Utils} from "../Utils/Utils";
+
+export class Connecter {
+    private readonly particle: Particle;
+    private readonly container: Container;
+
+    constructor(container: Container, particle: Particle) {
+        this.container = container;
+        this.particle = particle;
+    }
+
+    /**
+     * Connecting particles on hover interactivity
+     */
+    public connect(destParticle: Particle) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'connect' to have a typedef](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5098130)